LICENSE

```diff
@@ -1,6 +1,7 @@
 MIT License
 
-Copyright (c) 2017-2018 Monero Integrations
+Copyright (c) 2018, Ryo Currency Project
```

Comment:
Remove. You are free to contribute, respecting the license.

Reply:
@mosu-forge Don't do that, you don't want certain people suing you =)

Reply:
If this code is merged, then he is free to attribute his copyright to whatever project he wishes. If this is merged, then this must stay.

Reply:
```
Copyright (c) 2018, Ryo Currency Project
Portions Copyright (c) 2017-2018, Monero Integrations
```

may be reworded to:

```
Copyright (c) 2018, Ryo Currency Project
Copyright (c) 2017-2018, Monero Integrations
```

i.e. removing the word "Portions". I also do not care if Monero is moved above (c) Ryo Currency. This work was done under the funding of Ryo, so I would like for it to stay in the LICENSE file.